misc/css_generator.py

```diff
@@ -11,9 +11,10 @@
 """
 
 import os
+import mmap
 
-# File the CSS code will be written in
-dest_file = "to_be_appended_into_innovation.css"
+# File the SCSS code will be written in
+dest_file = "..\innovationlordmcfuzz.scss"

```

Comment:
I see lordmcfuzz in here. Does this mean that the rename script wasn't able to change this?

Reply:
I missed that in the readme and realize it is probably not great. I'm looking into eliminating the py script so it won't matter.

But you are correct, likely because it is in a file and not renaming the file itself.

Reply:
No more py